Mathlib/Analysis/Calculus/FDeriv/Norm.lean

```diff
@@ -0,0 +1,202 @@
+/-
+Copyright (c) 2024 Etienne Marion. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Etienne Marion
+-/
+import Mathlib.Analysis.Calculus.Deriv.Abs
+import Mathlib.Analysis.Calculus.LineDeriv.Basic
+
+/-!
+# Differentiability of the norm in a real normed vector space
+
+This file provides basic results about the differentiability of the norm in a real vector space.
+Most are of the following kind: if the norm has some differentiability property
+(`DifferentiableAt`, `ContDiffAt`, `HasStrictFDerivAt`, `HasFDerivAt`) at `x`, then so it has
+at `t • x` when `t ≠ 0`.
+
+## Main statements
+
+* `ContDiffAt.contDiffAt_norm_smul`: If the norm is continuously differentiable up to order `n`
+  at `x`, then so it is at `t • x` when `t ≠ 0`.
+* `differentiableAt_norm_smul`: If `t ≠ 0`, the norm is differentiable at `x` if and only if
+  it is at `t • x`.
+* `HasStrictFDerivAt.hasStrictFDerivAt_norm_smul`: If the norm has a strict Fréchet derivative
+  `f` at `x` and `t ≠ 0`, then it has `(SignType t) • f` as a strict Fréchet derivative at `t · x`.
```

Comment:
```suggestion
```
I'd remove this one, because I don't expect it will be as useful as the other ones you mention around here.